support-frontend/assets/pages/paper-subscription-landing/components/paperPrices.tsx

```diff
@@ -122,7 +126,7 @@ const getPlans = (
 	productPrices: ProductPrices,
 ) =>
 	ActivePaperProductTypes.map((productOption) => {
-		const priceAfterPromosApplied = finalPrice(
+		const priceAfterPromosApplied = getPriceWithDiscount(
```

Comment:
In `paperProductPrices.ts` the `finalPrice` and `getPriceWithDiscount` functions performed the same action. The duplicate `finalPrice` function has been removed from that file and the function being used here has been renamed

Reply:
This one makes me a tad nervous as it's a little confusing tracing the logic in those functions. I had a go and I'm fairly confident that they do the same thing so it's fine to remove the other one (I know there's a comment that says they're the same, but I never trust comments!).